### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by adeclosset**
Hard to make better without some more user testing

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -15,9 +15,10 @@
 
 <div class="container">
   <%= render 'layouts/alerts' %>
-	<%= yield %>
```
Comment (octaveauger) [inline app/views/layouts/application.html.erb:18]: @adeclosset why did you move it out of the container? I assume this is for the homepage image but it feels like a bad idea

Comment (adeclosset) [inline app/views/layouts/application.html.erb:18]: Yes it's for the jumbotron. If you put a container, it can't be full width. I've added containers on other pages.

Comment (octaveauger) [inline app/views/layouts/application.html.erb:18]: @adeclosset I've cleaned it up - better to exclude the container if it's the home page and not have to copy/paste what should be in the layout in every single view. Same result in the end.

It'd be neat to finish it up with a better logo (or worse case just "Passiton" in a bigger font) rather than the horrible black arrow I put as a placeholder early on. Makes sense? We can merge afterwards, or if you don't have time now I can merge and you do another PR for it.
